src/services/Store/store/membership.slice.ts

```diff
@@ -27,7 +27,7 @@ const slice = createSlice({
     setMembership(state, action: PayloadAction<MembershipStatus>) {
       state.record.push(action.payload);
     },
-    deleteMembership(state, action: PayloadAction<TAddress>) {
+    deleteMembership(state, action: PayloadAction<string>) {
```

Comment:
why changing this to a string ?

Reply:
Because `Brand` is causing issues with TS :(

Reply:
Then go with `string` unless you wanna cast :)